app/src/main/java/org/fossasia/openevent/general/auth/LoginViewModel.kt

```diff
@@ -117,6 +119,12 @@ class LoginViewModel(
             Patterns.EMAIL_ADDRESS.matcher(email).matches()
     }
 
+    fun checkFields(email: String, password: String) {
+        mutableAreFieldsCorrect.value = email.isNotEmpty() &&
+            Patterns.EMAIL_ADDRESS.matcher(email).matches() &&
```

Comment:
If you are checking email in checkFiels method itself?

Reply:
because that controls forgot password view. I will integrate both.

Reply:
done